jablib/build.gradle.kts

```diff
@@ -421,7 +421,8 @@ tasks.test {
     jvmArgs = listOf(
         "-javaagent:${mockitoAgent.asPath}",
         "--add-opens", "java.base/jdk.internal.ref=org.apache.pdfbox.io",
-        "--add-opens", "java.base/java.nio=org.apache.pdfbox.io"
+        "--add-opens", "java.base/java.nio=org.apache.pdfbox.io",
+        "--add-reads", "org.xmlunit.matchers=java.xml"
```

Comment:
Not sure why this is required now - the tests worked before?

Reply:
my bad just simply added for local test, i'll roll back

Reply:
done

Reply:
done